src/app/services/outboundServices/outbound-call-reallocation.service.ts

```
@@ -113,13 +128,20 @@ export class OutboundReAllocationService {
+        return this._http.post(this.grievanceMoveToBinURL, data)
+            .map(this.extractData)
+            .catch(this.handleError);
+    }
+
     private extractData(response: Response) {
         if (response.json().data) {
             return response.json().data;
         } else {
             return Observable.throw(response.json());
         };
     }
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)